src/encoded/tests/conftest.py

```diff
@@ -199,3 +203,52 @@ def embed_testapp(app):
 @pytest.fixture
 def wsgi_app(wsgi_server):
     return webtest.TestApp(wsgi_server)
+
+
+def _check_server_is_up(output):
+    """ Polls the given output file to detect
+
+        :args output: file to which server is piping out
+        :returns: True if server is up, False if failed
+    """
+    tries = 5
+    while tries > 0:
+        output.seek(0)  # should be first thing to be output.
+        out = output.read()
+        if 'Running' in out.decode('utf-8'):
+            return True
+        tries -= 1
+        time.sleep(1)  # give it a sec
+    return False
+
+
+@pytest.yield_fixture(scope='session', autouse=True)
+def start_moto_server_sqs():
+    """
+    Spins off a moto server running sqs, yields to the tests and cleans up.
+    """
+    delete_sqs_url = 'SQS_URL' not in os.environ
```

Comment:
I have better technology for doing this delete thing in another PR I'll submit soon, but this will suffice for now.